.github/workflows/nightly-release.yaml

```diff
@@ -68,8 +68,19 @@ jobs:
       - name: build toolchain
         run: |
           TARGET_TUPLE=($(echo ${{ matrix.target }} | tr "-" "\n"))
-          ./configure --prefix=/opt/riscv --with-arch=${TARGET_TUPLE[0]} --with-abi=${TARGET_TUPLE[1]}
-          sudo make -j $(nproc) ${{ matrix.mode }}
+          BUILD_TOOLCHAIN="./configure --prefix=/opt/riscv --with-arch=${TARGET_TUPLE[0]} --with-abi=${TARGET_TUPLE[1]}"
+          if [ "${{ matrix.mode }}" == "linux" ]; then # build toolchain with llvm
+            $BUILD_TOOLCHAIN --enable-llvm --enable-linux
+            sudo make -j $(nproc) all build-sim SIM=qemu
```

Comment:
Drop `build-sim SIM=qemu`, it build in next step

Reply:
Thanks, it's done.